### PR TITLE
Make test dependency clearer

### PR DESCRIPTION
**Pull request by freethan**
README clarifications re selenium-webdriver and Node minimum version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,21 +20,23 @@ For extended documentation see [here](https://developer.here.com/rest-apis/docum
...
+`npm run server`
 
 A static http server will be started on port 5000, i.e. [http://localhost:5000](http://localhost:5000)
 
 ### Tests
 
 Make sure your server is running and execute
 
-    npm run test
+`npm run test`
```
Comment (sschuberth) [inline README.md]: Just `npm test`?

Comment (freethan) [reply README.md]: Sure, no problem. It's one of the default targets.

File: README.md [hunk 2]
```diff
@@ -20,21 +20,23 @@ For extended documentation see [here](https://developer.here.com/rest-apis/docum
 
 ### Installation
 
-Make sure you have already installed the latest version of Node (at least version 4.2) and NPM (Node Package Manager) and then run:
-
-    npm install
+Make sure you have already installed the latest version of `Node` (at least version 6.10.3) and `NPM` (Node Package Manager) and then run:
```
Comment (sschuberth) [inline README.md]: Strictly speaking, I would not use `code` formatting here, as you're referring to the literal name of the apps, not to the name of the command line executables. So, IMO, either use Node and NPM, or `node` and `npm` (lower case).

File: README.md [hunk 3]
```diff
@@ -20,21 +20,23 @@ For extended documentation see [here](https://developer.here.com/rest-apis/docum
...
 
+`npm install`
```
Comment (sschuberth) [inline README.md]: I don't really see a reason for this change as from a visual appearance perspective they look the same in the end: `backticks` should be used for inline code, and

    four spaces indentation

for code blocks.

**Review by sschuberth — CHANGES_REQUESTED** [review]

**Review by sschuberth — APPROVED** [review]

**Pull request merged**